### PR TITLE
[zh] Fix `Kubernetes` typo for docs

### PR DESCRIPTION
**Pull request by Sea-n**
- Fix typos like `Kuberbetes`, `Kubernets`, `Kubernates`, `Kubernete`
- Re-sync some files

### CODE REVIEW

File: content/zh/docs/concepts/storage/projected-volumes.md [hunk 1]
```diff
@@ -49,10 +49,11 @@ Currently, the following types of volume sources can be projected:
 
 <!--
 All sources are required to be in the same namespace as the Pod. For more details,
-see the [all-in-one volume design document](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/all-in-one-volume.md).
+see the [all-in-one volume](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/all-in-one-volume.md).
```
Comment (howieyuen) [inline content/zh/docs/concepts/storage/projected-volumes.md:52]: delete this line

**Review by howieyuen — CHANGES_REQUESTED** [review]
nice work except  a nit

File: content/zh/docs/concepts/storage/projected-volumes.md (hunk 1 above)
Comment (Sea-n) [reply content/zh/docs/concepts/storage/projected-volumes.md]: How could I missed that!

Thanks very much, fixed. 👍

**Pull request merged**